src/app/panel/page.tsx

```diff
@@ -0,0 +1,10 @@
+"use client";
+
+export default function Home() {
+
+  return (
+    <main>
+      <h1>Controle Panel</h1>
```

Comment:
Is this intentional? `Controle` and not `Control`?

Reply:
Dude, fuck off :)